public/app/percona/shared/helpers/payloadToCamelCase.ts

```diff
@@ -0,0 +1,35 @@
+// Recursively converts object keys to camelCase. Useful for payload conversions
+// Usage: <CamelCasedPayloadType>payloadToCamelCase(payload)
+export const payloadToCamelCase = <T extends object, R extends object>(object: T, ignoredKeys: string[] = []): R => {
+  // We use an outer function just to help with typings
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const recursive = (o: any): any => {
+    if (o === Object(o) && !Array.isArray(o) && typeof o !== 'function') {
+      const n = {};
```

Comment:
Maybe a little bit more clear solution could be to add here const n: Record<any, any> = {}; and get rid of ts-ignore